CartPole/state_utilities.py

```diff
@@ -28,6 +28,16 @@
 STATE_INDICES = {x: np.where(STATE_VARIABLES==x)[0][0] for x in STATE_VARIABLES}
 STATE_INDICES_REDUCED = {x: np.where(STATE_VARIABLES_REDUCED==x)[0][0] for x in STATE_VARIABLES_REDUCED}
 
+"""Define indices of values in state statically"""
```

Comment:
Since these indices are constants, they are now exported as such. No need to call the cartpole_state_varname_to_index, which takes more time to run.